### PR TITLE
returning default zoom value if the route progress state is uncertain

### PR DESCRIPTION
**Pull request by cafesilencio**
#2634 Description

When the route progress is uncertain the camera should zoom out to the default zoom level.

### CODE REVIEW

File: libnavigation-ui/src/main/java/com/mapbox/navigation/ui/puck/NavigationPuckPresenter.kt [hunk 1]
```diff
@@ -56,12 +57,14 @@ class NavigationPuckPresenter(private val mapboxMap: MapboxMap, puckDrawableSupp
...
 
     private fun updateCurrentLocationDrawable(@DrawableRes gpsDrawable: Int) {
+        val cameraUpdateWithPadding = CameraUpdateFactory.paddingTo(mapboxMap.cameraPosition.padding)
         val options: LocationComponentOptions = mapboxMap.locationComponent.locationComponentOptions
         if (options.gpsDrawable() != gpsDrawable) {
             val newOptions = options.toBuilder()
-                    .gpsDrawable(gpsDrawable)
-                    .build()
+                .gpsDrawable(gpsDrawable)
+                .build()
```
Comment (Guardiola31337) [inline libnavigation-ui/src/main/java/com/mapbox/navigation/ui/puck/NavigationPuckPresenter.kt]: NIT unnecessary formatting changes

Comment (cafesilencio) [reply libnavigation-ui/src/main/java/com/mapbox/navigation/ui/puck/NavigationPuckPresenter.kt]: NavigationCamera uses DynamicCamera
